src/bundler/bundle_v2.zig

```diff
@@ -3027,8 +3299,10 @@ pub const ParseTask = struct {
         opts.features.emit_decorator_metadata = bundler.options.emit_decorator_metadata;
         opts.features.unwrap_commonjs_packages = bundler.options.unwrap_commonjs_packages;
         opts.features.hot_module_reloading = bundler.options.output_format == .internal_kit_dev and !source.index.isRuntime();
-        opts.features.react_fast_refresh = (bundler.options.hot_module_reloading or bundler.options.react_fast_refresh) and
-            loader.isJSX() and !source.path.isNodeModule();
+        opts.features.react_fast_refresh = target == .browser and
```

Comment:
only browser?

Reply:
Yes. there is no mounted DOM to live patch on the server.